assets/src/block-validation/error/index.js

```diff
@@ -70,15 +69,14 @@ export function Error( { clientId, status, term_id: termId, ...props } ) {
 							{ __( 'Select block', 'amp' ) }
 						</Button>
 					) }
-					<a
+					<ExternalLink
 						href={ detailsUrl.href }
 						target="_blank"
```

Comment:
Instead of `_blank` we may want this to be a consistent name like `validated-url-post-${ postId }`  so that each of the View Details links will be routed to the same tab.